Sources/UIApplication+handleSDLEvents.swift

```diff
@@ -67,7 +67,8 @@ extension UIApplication {
 
                 if keyModifier.contains(KMOD_LGUI) || keyModifier.contains(KMOD_RGUI) {
                     if e.key.keysym.sym == 114 { // CMD-R
-                        UIApplication.restart()
+                        UIScreen.main = nil
+                        UIScreen.main = UIScreen()
```

Comment:
Does it make a difference if we remove `UIScreen.main = nil`. Setting a new instance instantly should also deinit the old uiscreen instance

Reply:
The order is important. We have to deinit the old one first